src/xclim/indicators/atmos/_precip.py

```diff
@@ -815,3 +816,15 @@ class HrPrecip(Hourly):
     cell_methods="time: sum over days",
     compute=indices.water_cycle_intensity,
 )
+
+antecedent_precipitation_index = Indicator(
+    title="Antecedent Precipitation Index",
```

Comment:
```suggestion
```
The `title` is taken from the first line of the docstring, no need to repeat it here.